decidim/decidim
# Fix Stack Level Too Deep on request committee member in initiatives

#### :tophat: What? Why?
This PR fixes a Stack Level too deep error that happens on committee members in intiatives.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #12126 

#### Testing
1. The pipeline should be green
2. On develop branch patch committee_requests_controller_spec.rb as done in this PR and run the spec 
3. See the stack too deep error 
4. Apply full patch, everything should pass
5. Using the reproduction steps described in #12126 
6. Apply fix 
7. See everything is changed

:hearts: Thank you!
